mava/systems/jax/config.py

```diff
@@ -189,3 +190,32 @@ def get(self) -> SimpleNamespace:
             )
 
         return SimpleNamespace(**self._built_config)
+
+    def get_local_config(self, component: Type[Component]) -> SimpleNamespace:
+        """Get built config for a single component.
+
+        Args:
+            component: component to provide config for.
+
+        Returns:
+            built config for a single component.
+        """
+        if not self._built:
+            raise Exception(
+                "The config must first be built using .build()"
+                "before calling .get_local_config()."
+            )
+
+        global_config = self._built_config
+        local_config: Dict[str, Any] = {}
+
+        config_class = component.config_class()
+        if not config_class:  # no config class for component
+            return SimpleNamespace()
+
+        # Check global config for names which appear in the config class
+        for parameter_name, value in global_config.items():
+            if hasattr(config_class, parameter_name):
+                local_config[parameter_name] = global_config[parameter_name]
+
+        return SimpleNamespace(**local_config)
```

Comment:
Would `component.config_class(**local_config)` work, cause then the types will even be the same?

Reply:
Nope, but `component.config_class()(**local_config)` works, and I have made the change